mixed-content/generic/worker.js

```diff
@@ -0,0 +1,7 @@
+try {
+  // Works on same origin.
+  postMessage('done');
+} catch(err) {
+  // Works cross-origin.
+  postMessage('done', '*');
```

Comment:
1. I don't believe it's possible to load an HTTPS worker on an HTTP page (see step #5 of https://html.spec.whatwg.org/multipage/workers.html#dom-worker (note that Firefox allows `data:` Workers, but it also treats `data:` as the same-origin as its embedder; Chrome does not, and does not)).

2. Regardless, a worker's `postMessage` always talks to its embedder; it doesn't have an origin flag (the second argument is a set of objects to transfer from one context to the other: see https://html.spec.whatwg.org/multipage/workers.html#dedicated-workers-and-the-dedicatedworkerglobalscope-interface:dom-dedicatedworkerglobalscope-postmessage).

You can simplify this file to `postMessage('done');`.